deeppavlov/core/agent_v2/postprocessor.py

```diff
@@ -0,0 +1,37 @@
+from typing import Sequence
+from abc import abstractmethod, ABCMeta
+
+from deeppavlov.models.preprocessors.person_normalizer import PersonNormalizer
+from deeppavlov.models.tokenizers.utils import detokenize
+
+
+class Postprocessor(metaclass=ABCMeta):
+
+    @abstractmethod
+    def __call__(self, state: dict) -> Sequence[str]:
+        raise NotImplementedError
+
+
+class DefaultPostprocessor(Postprocessor):
+    def __init__(self) -> None:
+        self.person_normalizer = PersonNormalizer(per_tag='PER')
+
+    def __call__(self, state: dict) -> Sequence[str]:
+        new_responses = []
+        for d_state in state['dialogs']:
+            # get tokens & tags
+            response = d_state['utterances'][-1]
+            ner_annotations = response['annotations']['ner']
```

Comment:
There is a `DialogsParser` class at DeepPavlov, which can be used for parsing states. If there is not enough functionality in this class, we can add more methods to it. 
We should make a server from `Postprocessor` and parse state with `DialogParser`